src/styles/pages/troubleshooting-and-common-issues.scss

```diff
@@ -0,0 +1,49 @@
+$arrow-right-base64: "data:image/png;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz4KPHN2ZyB3aWR0aD0iN3B4IiBoZWlnaHQ9IjEycHgiIHZpZXdCb3g9IjAgMCA3IDEyIiB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiPgogICAgPCEtLSBHZW5lcmF0b3I6IFNrZXRjaCA1MC4yICg1NTA0NykgLSBodHRwOi8vd3d3LmJvaGVtaWFuY29kaW5nLmNvbS9za2V0Y2ggLS0+CiAgICA8dGl0bGU+YXJyb3dzL2Fycm93LXJpZ2h0PC90aXRsZT4KICAgIDxkZXNjPkNyZWF0ZWQgd2l0aCBTa2V0Y2guPC9kZXNjPgogICAgPGRlZnM+CiAgICAgICAgPHBhdGggZD0iTTExLjY1Njg4NDIsNy4yNDMwMjc5OCBDMTEuOTU3Nzg5NSw3LjU0MzkzMzI0IDEyLjAyNjg0MjQsNy45ODg4OTMwNyAxMS44NjQwNDMsOC4zNTU5MTQ3MyBDMTEuODE1NDYxNyw4LjQ2NTA5NTU0IDExLjc0NjQ0Miw4LjU2NzQxMzkyIDExLjY1Njg4NDIsOC42NTY5NzE2NCBMMTEuNjU2ODg0Miw4LjY1NzI0MTU0IEw2LjcwNzEzNjc3LDEzLjYwNjk4OSBDNi4zMTY2MTI0OCwxMy45OTc1MTMzIDUuNjgzNDQ3NSwxMy45OTc1MTMzIDUuMjkyOTIzMjEsMTMuNjA2OTg5IEw1LjI5Mjg2MzIzLDEzLjYwNjkyOSBDNC45MDIzMzg5NCwxMy4yMTY0MDQ3IDQuOTAyMzM4OTQsMTIuNTgzMjM5OCA1LjI5Mjg2MzIzLDEyLjE5MjcxNTUgTDkuNTM1NTc4ODksNy45NDk5OTk4MSBMNS4yOTI4NjMyMywzLjcwNzI4NDE1IEM0LjkwMjMzODk0LDMuMzE2NzU5ODYgNC45MDIzMzg5NCwyLjY4MzU5NDg4IDUuMjkyODYzMjMsMi4yOTMwNzA1OSBMNS4yOTI5MjMyMSwyLjI5MzAxMDYxIEM1LjY4MzQ0NzUsMS45MDI0ODYzMSA2LjMxNjYxMjQ4LDEuOTAyNDg2MzEgNi43MDcxMzY3NywyLjI5MzAxMDYxIEwxMS42NTY4ODQyLDcuMjQyNzU4MDcgTDExLjY1NzEwNzgsNy4yNDI5ODE3IEwxMS42NTY4ODQyLDcuMjQzMDI3OTggWiIgaWQ9InBhdGgtMSI+PC9wYXRoPgogICAgPC9kZWZzPgogICAgPGcgaWQ9IkRvY3VtZW50YXRpb24iIHN0cm9rZT0ibm9uZSIgc3Ryb2tlLXdpZHRoPSIxIiBmaWxsPSJub25lIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiPgogICAgICAgIDxnIGlkPSJEb2NzLUhvbWUtMiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoLTM3OS4wMDAwMDAsIC02MjAuMDAwMDAwKSI+CiAgICAgICAgICAgIDxnIGlkPSJhcnJvd3MvYXJyb3ctcmlnaHQiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDM3NC4wMDAwMDAsIDYxOC4wMDAwMDApIj4KICAgICAgICAgICAgICAgIDxnIGlkPSJpY29uL2Fycm93LXJpZ2h0Ij4KICAgICAgICAgICAgICAgICAgICA8cG9seWdvbiBpZD0iU2hhcGUiIHBvaW50cz0iMCAwIDE2IDAgMTYgMTYgMCAxNiI+PC9wb2x5Z29uPgogICAgICAgICAgICAgICAgICAgIDxtYXNrIGlkPSJtYXNrLTIiIGZpbGw9IndoaXRlIj4KICAgICAgICAgICAgICAgICAgICAgICAgPHVzZSB4bGluazpocmVmPSIjcGF0aC0xIj48L3VzZT4KICAgICAgICAgICAgICAgICAgICA8L21hc2s+CiAgICAgICAgICAgICAgICAgICAgPHVzZSBmaWxsPSIjRDhEOEQ4IiB4bGluazpocmVmPSIjcGF0aC0xIj48L3VzZT4KICAgICAgICAgICAgICAgICAgICA8ZyBpZD0ibmV1dHJhbC9AbjkwMC0oIzE2MTYxNikiIG1hc2s9InVybCgjbWFzay0yKSIgZmlsbD0iIzE2MTYxNiI+CiAgICAgICAgICAgICAgICAgICAgICAgIDxyZWN0IGlkPSJQYXRoIiB4PSIwIiB5PSIwIiB3aWR0aD0iMTYiIGhlaWdodD0iMTYiPjwvcmVjdD4KICAgICAgICAgICAgICAgICAgICA8L2c+CiAgICAgICAgICAgICAgICA8L2c+CiAgICAgICAgICAgIDwvZz4KICAgICAgICA8L2c+CiAgICA8L2c+Cjwvc3ZnPg==";
```

Comment:
is there a reason for inlined image (png), encoded in base64?

Reply:
Yep, I'm using base64 because I cannot access the `{{site.baseurl}}` variable in scss-imported-through-js; I don't want to hardcode links like `background-image: url("/docs/assets/etc/etc") as it will break in preview and if any urls ever change. I feel that for a small icon base64 is better, but if we ever run into having to put actual images as background-images in css we will need to find a better solution.